src/patterns-hmrc-govuk-team/service-availability-and-service-issues/index.njk

```diff
@@ -4,146 +4,113 @@ layout: twoColumnPage.njk
+
+## Service issues
+
+None.
+
+## Other services
+
+Check [availability and issues](https://www.gov.uk/government/collections/hmrc-service-availability-and-issues) for other services.',
+      canCopy: true
+    })
+  }}
+
+<p class="govuk-body">Use a call to action box if there is planned downtime or service issues.</p>
+
+<p class="govuk-body">Do not use 'midnight', use '11:59pm',</p>
 
 {{
-  govukTable({
-    caption: 'HTML attachment',
-    firstCellIsHeader: true,
-    head: [
-      {
-        text: 'Field'
-      },
-      {
-        text: 'What to enter'
-      }
-    ],
-    rows: [
-      [
-        {
-          text: 'Title'
-        },
-        {
-          html: '<p class="govuk-body">[Service]: service availability and issues</p>
-          <p class="govuk-body govuk-!-font-weight-bold">Example:</p>
-          <p>Corporation Tax: service availability and issues</p>'
-        }
-      ],
-      [
-        {
-          text: 'Body text'
-        },
-        {
-          html: '## Service availability
-          <p>Full service available.</p>
-          <p>Sign in to [insert service name] (add the url)</p>
-          <p>## Planned downtime</p>
-          <p>None.</p>
-          <p>## Service issues</p>
-          <p>None.</p>
-          <p>## Other services</p>
-          <p>Check [availability and issues] (<a href="https://www.gov.uk/government/collections/hmrc-service-availability-and-issues" class="govuk-link">https://www<wbr>.gov<wbr>.uk<wbr>/government<wbr>/collections<wbr>/hmrc-service-availability-and-issues</a>) for other services.</p>
-          <p>Use a call to action box when adding &#39Planned downtime&#39.</p>
-          <p>$CTA</p>
-          <p>Sunday 26 September 2021 11:59pm to 3am Monday 27 September 2021</p>
-          <p>$CTA</p>
-          <p>Due to scheduled maintenance you will be unable to access our online service from 11:59pm on Sunday 26 September 2021 to 3am on Monday 27 April 2021.</p>
-          <p>We apologise for any inconvenience this may cause.</p>
-          <p>Add &#39None.&#39 when there is no planned downtime or service issues.</p>
-          <p>Note: Do not use &#39midnight&#39, use 11:59pm</p>'
-        }
-      ],
-      [
-        {
-          text: 'Public change note'
-        },
-        {
-          html: '<p class="govuk-body govuk-!-font-weight-bold">Service availability</p>
-<p class="govuk-body govuk-!-font-weight-bold">Examples:</p>
+    codeSnippet({
+      code: '$CTA
+
+Sunday 26 September 2021 11:59pm to 3am Monday 27 September 2021
+
+$CTA
+
+Due to scheduled maintenance you will be unable to access our online service from 11:59pm on Sunday 26 September 2021 to 3am on Monday 27 April 2021.
+
+We apologise for any inconvenience this may cause.',
+      canCopy: true
+    })
+  }}
+
+<h3 class="govuk-heading-m">Public change note</h3>
+
+<h4 class="govuk-heading-s">Service availability</h4>
+
+<p class="govuk-body">Examples:</p>
 <p><ol class="govuk-list govuk-list--number">
```

Comment:
Should this be a `<ul>`? It doesn't look like these have a specific order. Also, an `<ol>` shouldn't be inside a `<p>`

Reply:
Fixed.